rplugin/python3/deoplete/sources/ternjs.py

```diff
@@ -314,7 +303,8 @@ def completation(self, pos):
             for rec in data["completions"]:
                 completions.append({
                     "word": rec["name"],
-                    "menu": self.completion_icon(rec.get("type")),
+                    "kind": self.completion_icon(rec.get("type")),
+                    "abbr": rec.get("type").replace('fn', rec["name"], 1),
```

Comment:
@carlitux actually, maybe we should use a regex here? to only replace 'fn' if it begins the string. can you add that? I don't really know python.